blueocean-dashboard/src/main/js/components/MultiBranch.jsx

```diff
@@ -48,54 +49,44 @@ EmptyState.propTypes = {
 };
 
 export class MultiBranch extends Component {
-    constructor(props) {
```

Comment:
you do the same refactor later on again, why do not use OrganizationPipelines and pass _isUnsupportedJob in the props to the component?